### DISCUSSION TITLE
PrimeNG docs for `p-table` - Template example missing crucial property `[first]`

### DISCUSSION THREAD

**Discussion by yenmangu** [primefaces/community#1315] [PrimeNG] [answered]
In the docs for the `p-table`, where it shows how to implement programmatic pagination buttons [here](https://primeng.org/table#paginator-programmaticl),
 it states to bind to the `first` property, however the template code provided does actually include that. 

```
<div class="card">
    <p-table
        [value]="customers"
        [paginator]="true"
        [rows]="5"
        [showCurrentPageReport]="true"
        [tableStyle]="{ 'min-width': '50rem' }"
        currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries"
        (onPage)="pageChange($event)"
        [rowsPerPageOptions]="[10, 25, 50]"
    >
```

As far as I can see, without specifically including

```
[first]="first"
```
The custom buttons will not work, as they are not changing the `[first]` property of the table.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue, we'll update it in the next release.
